packages/squiggle-lang/src/public/SqValue/SqLambda.ts

```diff
@@ -19,22 +20,25 @@ export type SqLambdaParameter = {
 type SqLambdaSignature = SqLambdaParameter[];
```

Comment:
This type will evolve further; we now have `TTypedLambda`, maybe we should expose it directly.